### PR TITLE
A big rename from GoTermAdventure to TermAdventure

### PR DESCRIPTION
**Pull request by vidriduch**
Renaming of everything from GoTermAdvenutre (GTA) to just TermAdventure (TA)

Signed-off-by: Ondrej Jariabka <o.jariabka@gmail.com>

### CODE REVIEW

**Comment by mrshu** [comment]
Looks good, thanks @vidriduch!

**Comment by mrshu** [comment]
Just a note: we should probably rename the repository as well

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# go-term-adventure
+# term-adventure
```
Comment (Adman) [inline README.md:1]: adventure or adventures?

Comment (vidriduch) [reply README.md:1]: I think we settled on `adventure`

**Pull request merged**